util/example/tlgen/xbar_2x2.hjson

```diff
@@ -1,11 +1,48 @@
 {
   name: "2x2",
-  clock: "main",
+  clock: "clk_main_i",
+  clock_connections: {
+    clk_main_i: main
```

Comment:
There is a combination of styles, primarily `key: value` and `key: "value"` and `key: "value",`

Probably worth unifying in this hjson file. The style guide says this:

```
In some cases Hjson allows the quotes to be omitted from values,
but this is not recommended because the value string will not be
terminated by a comma so there is potential for confusion when
multiple key-value pairs are put on the same line.
```

It says the trailing comma is not unified:

```
Hjson allows commas to be omitted when an item that would
be terminated with a comma is terminated by the end of a line,
and will tolerate extra “trailing” commas. There is currently no
style-guide rule on this. In general, use the comma if it is likely
items may be combined on a single line and ommit [sic] it if it
keeps the file cleaner.
```

Reply:
Thanks! revised now.